Cargo.toml

```diff
@@ -35,6 +35,8 @@ tokio = { version = "1.15.0", features = [
 tracing = "0.1.35"
 tracing-subscriber = { version = "0.3.17", features = ["env-filter"] }
 rust-strictmath = "0.1.1"
+serde = { version = "1.0", features = ["derive"] }
+serde_yaml = "0.8"
```

Comment:
User profiles are broadly used in SCT as this is the only mode that allows to define custom schemas. Simple modes, such as `write/read` always create table with only `blob` columns.

OTOH, some users may only want to perform simple benchmarks (for these, `write/read` workloads are enough) - it would make sense to hide user profiles behind a feature then.